docs/tutorial/5-relationships-and-hyperlinked-apis.md

```diff
@@ -94,6 +94,16 @@ Notice that we've also added a new `'highlight'` field.  This field is of the sa
 
 Because we've included format suffixed URLs such as `'.json'`, we also need to indicate on the `highlight` field that any format suffixed hyperlinks it returns should use the `'.html'` suffix.
 
+**Important:**  
+When you are manually instantiating these serializers inside your views (e.g., in `SnippetDetail` or `SnippetList`), you **must** pass `context={'request': request}` so the serializer knows how to build absolute URLs.
```

Comment:
I believe that by this point, the `SnippetDetail` and `SnippetList` both inherit `GenericAPIView`, which has a `get_serializer_context()` method. Perhaps this would be a good moment to mention this method.

Reply:
Yes, perhaps. What I'm suggesting is to add a reminder that if you use one of the generic views, you have a method to build the context data that you should use instead of creating the whole dict literal